### ISSUE TITLE
Menubar placement in WP

### ISSUE THREAD

**Issue by colemanw** [aydun/uk.squiffle.kam#20]
Currently the CiviCRM menu covers up the WP top navigation menu. This may not be necessary or desirable. Discuss...

**Comment by aydun** [comment]
One consideration for the placement of menus is the impact on accessibility and use with screen readers.  I think hiding other menus and just displaying the Civi one along with an option to toggle back to the CRM menus may be better as it reduces the clutter - but I'm no expert.  I'll flag this in the ~accessibility channel.

**Comment by JoeMurray** [comment]
tl;dr toggling between cms and civi menus is good enough imho

In Drupal default install, the first menu dropdown for the CiviCRM menu has an option to 'Hide' the CiviCRM menu which in a standard Drupal install will reveal a Drupal menu that had been hidden. It also creates popup to restore the CiviCRM menu. The popup does not last if one navigates away from the page. 

WCAG is concerned with perceivability of functionality. However, so long as our menu allows user to get back to main (CMS) menu, and the main (CMS) menu allows user to get to CiviCRM then I think we're okay from an accessibility compliance perspective. 

Whether hiding the main menu is a good usability practice including from the perspective of accessibility is a good question. I think it is a reasonable choice as the tendency is to have long sessions of work within CiviCRM's backend context, not regular back and forth between CiviCRM and the CMS. There is an exception in WordPress, where after creating CiviCRM content for public exposure it is necessary to go over to WP to create / update content to include a shortcode to expose CiviCRM content to the public.

**Comment by colemanw** [comment]
I do think that for the sake of accessibility *hiding* the CMS menu with `display: none` is better than just covering it up with an aggressive z-index. I think for both of these problems (placement in WP & Joomla, hiding the menu in Drupal) it's ok to add some cms-specific css & js files. We can try to make this menu code as generic as possible but a few tweaks for each cms are acceptable & beneficial. I'll roll up a PR for that.

**Comment by colemanw** [comment]
Well it's not too hard to achieve what @christianwach suggested... here it is.
What I'm not sure of is whether or not this is an actual improvement. 2 menubars takes up more screen space, the top WP bar isn't really doing much, and there is the "hide menu" option on the CiviCRM nav bar already.

![screenshot from 2018-11-22 08-39-48](https://user-images.githubusercontent.com/2874912/48906461-83b12400-ee32-11e8-8a38-fb9e2bb0f4d4.png)

**Comment by christianwach** [comment]
I'm not a fan of obscuring the native WordPress menu, but the alternatives seem worse. There was discussion on [a CAU issue](https://github.com/christianwach/civicrm-admin-utilities/issues/1) a few years ago - no conclusion reached however. I'd vote for keeping as is for the moment.

**Comment by colemanw** [comment]
Yes that's what I've concluded as well.

**Comment by colemanw** [comment]
@aydun @christianwach @JoeMurray I've decided to let the user choose instead of us choosing for them:
https://github.com/aydun/uk.squiffle.kam/pull/21
